paymentsheet/src/main/java/com/stripe/android/common/analytics/experiment/LogLinkGlobalHoldbackExposure.kt

```diff
@@ -0,0 +1,29 @@
+package com.stripe.android.common.analytics.experiment
+
+import com.stripe.android.common.analytics.experiment.ExperimentGroup.CONTROL
+import com.stripe.android.common.analytics.experiment.ExperimentGroup.TREATMENT
+import com.stripe.android.common.analytics.experiment.LoggableExperiment.LinkGlobalHoldback
+import com.stripe.android.core.utils.FeatureFlags
+import com.stripe.android.model.ElementsSession
+import com.stripe.android.paymentsheet.analytics.EventReporter
+import javax.inject.Inject
+
+internal class LogLinkGlobalHoldbackExposure @Inject constructor(
+    private val eventReporter: EventReporter
+) {
+
+    operator fun invoke(
+        elementsSession: ElementsSession,
+    ) = runCatching {
+        if (FeatureFlags.linkGlobalHoldbackExposureEnabled.isEnabled) {
+            val experimentsData = requireNotNull(elementsSession.experimentsData)
```

Comment:
This feels odd. We're using runCatching + requireNotNull. I think it would be easier to reason about what's happening here if we didn't use exceptions.